ko-KR/index.html

```diff
@@ -0,0 +1,256 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Servo, 병렬 브라우저 엔진</title>
+  <link rel='stylesheet' href='css/bootstrap.css'/>
```

Comment:
I suspect this css href will fail. `/css/bootstrap.css` may work

Reply:
yeah exactly. I edited it.